src/pal/tests/CMakeLists.txt

```diff
@@ -23,7 +27,7 @@ elseif(PAL_CMAKE_PLATFORM_ARCH_ARM64)
   add_definitions(-DBIT64=1)
   add_definitions(-D_WIN64=1)
 else()
-  message(FATAL_ERROR "Only ARM, ARM64 and AMD64 is supported")
+  message(FATAL_ERROR "Only ARM, ARM64, I386, and AMD64 is supported")
```

Comment:
We already use CLR_CMAKE_TARGET_ARCH_I386 in the CMakeLists.txt, so this matches the change.